tests/unit/mathematics/linalg/SGLinalg_unittest.cc

```diff
@@ -38,11 +38,10 @@ TEST(LinalgRefactor, linalg_gpu_backend_dot_without_gpu_backend)
 	EXPECT_NEAR(result, 20, 1E-15);
 }
 
+#ifdef HAVE_VIENNACL
```

Comment:
this should be implemented in a different file i reckon... like `SGLinalg_gpu...`